arch/arm/core/cortex_m/irq_manage.c

```diff
@@ -92,6 +96,26 @@ void z_arm_irq_priority_set(unsigned int irq, unsigned int prio, uint32_t flags)
 	NVIC_SetPriority((IRQn_Type)irq, prio);
 }
 
+#if !defined(CONFIG_ARM_CUSTOM_INTERRUPT_CONTROLLER)
+void arch_irq_enable(unsigned int irq)
+{
+	arm_irq_enable(irq);
+}
+
+void arch_irq_disable(unsigned int irq)
+{
+	arm_irq_disable(irq);
+}
+
+int arch_irq_is_enabled(unsigned int irq)
+{
+	return arm_irq_is_enabled(irq);
+}
+
+void z_arm_irq_priority_set(unsigned int irq, unsigned int prio, uint32_t flags)
+{
+	arm_irq_priority_set(irq, prio, flags);
+}
```

Comment:
Where are those defined in such case that `CONFIG_ARM_CUSTOM_INTERRUPT_CONTROLLER` is defined ?

Reply:
In my understanding, It is not clear because no practical case of using ARM_CUSTOM_INTERRUPT_CONTROLLER.
(Only this test case uses it.)

The comments assume that this is meant for the case of using an alternative controller for the NVIC.
So my understanding is that a custom interrupt controller must implement these functions alternatively.